src/BlazorWasmSdk/Tasks/GenerateBlazorWebAssemblyBootJson.cs

```diff
@@ -1,8 +1,10 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
```

Comment:
```suggestion
```

Reply:
Tooling actually generates the extra `//` so I'm going to leave it there (I have a separate PR where I apply the correct license to all our files)